src/bp/migrations/v12_0_0-1560882898-move_users_to_db.ts

```diff
@@ -34,6 +34,7 @@ const migration: Migration = {
         await userRepo.createUser({
           email: user.email,
           strategy: 'default',
+          tokenVersion: 1,
```

Comment:
I'm not sure I understand why you would want to edit older migration?

Reply:
I've changed the typings, so I need to update them. And if a user migrate from 12.0.0 to the latest one, he'll need that field to be filled